packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -79,19 +84,38 @@ const MyProfile = observer(() => {
                                 <div className='my-profile__stats-cell-info'>{total_orders_count || '-'}</div>
                             </Table.Cell>
                             <div className='my-profile__stats-cell-separator' />
-                            <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>
-                                    {localize('Buy ({{currency}})', { currency: general_store.client.currency })}
-                                </div>
-                                <div className='my-profile__stats-cell-info'>{buy_orders_count || '-'}</div>
-                            </Table.Cell>
-                            <div className='my-profile__stats-cell-separator' />
-                            <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>
-                                    {localize('Sell ({{currency}})', { currency: general_store.client.currency })}
-                                </div>
-                                <div className='my-profile__stats-cell-info'>{sell_orders_count || '-'}</div>
-                            </Table.Cell>
+                            {general_store.props.is_mobile ? (
```

Comment:
Please use `isMobile` from `@deriv/shared`, we're moving to that instead of using props.